app/models/container_quota_item.rb

```diff
@@ -1,3 +1,15 @@
 class ContainerQuotaItem < ApplicationRecord
   belongs_to :container_quota
+
+  def quota_desired
+    quota_desired_int || quota_desired_float
```

Comment:
one of them is always nil while the other contains the actual quota value (which can be either int or float)